src/App.js

```diff
@@ -1,11 +1,31 @@
 import React from 'react'
+import { Switch, Route, Redirect } from 'react-router'
+import { Layout } from 'antd'
 
-import AppLayout from './core/components/layout'
+import { Content } from 'common/ui-kit'
+import { Header, Footer } from 'core/components'
+import { PortalPage, LoginPage, NotFoundPage } from 'pages/index'
 
-import './App.scss'
+import './App.css'
+
+const isLogged = true
 
 function App() {
-  return <AppLayout />
+  return (
+    <Layout>
+      <Header />
+      <Content>
+        <Switch>
+          <Route exact path="/login" component={LoginPage} />
+          <Route exact path="/404" component={NotFoundPage} />
```

Comment:
export all patch from constant file

Reply:
it's not final version, i think it's not obligios now